application/utils.py

```diff
@@ -0,0 +1,44 @@
+from constans.constans import EAST, NORTH, SOUTH, WEST
+from constans.constants_game import LARGE
+from game.board import Board
+from game.character import Character
+from game.player import Player
+
+
+def is_a_player_character(row, col, current_player: Player, board: Board) -> bool:
+    cell = board.get_cell(row, col)
+    character: Character = cell.character
+    return character.player.user_name == current_player.user_name if character else False
+
+
+def is_frendly_fire(row, col, direction, current_player: Player, board: Board) -> bool:
+    target_row, target_col = target_position_within_bounds(row, col, direction)
+    target_cell = board.get_cell(target_row, target_col)
+    if (
+        target_cell.character is not None and
```

Comment:
As far as i know: 
`if target_cell.character is not None and ...`
Can be replaced by 
`if target_cell.character and ...`

Reply:
Great, I fixed in [66a45ef](https://github.com/evbeda/edagames-wumpus/commit/66a45eff499eafebbc87846fc39f99ba60d42d59) commit. Thanks for the feedback!